shared/lib_irradproc.cpp

```diff
@@ -1090,12 +1106,25 @@ int irrad::calc()
 	double t_sunrise = sunAnglesRadians[4];
 	double t_sunset = sunAnglesRadians[5];
 
+	if (t_sunset > 24) //sunset is legitimately the next day, so recalculate sunset from the previous day
+	{
+		double sunanglestemp[9];
+		solarpos(year, month, day - 1, 12, 0.0, latitudeDegrees, longitudeDegrees, timezone, sunanglestemp);
```

Comment:
What happens if the timestep is 3/1/2018 and (day -1 = 0)?

Reply:
Ah, good point! I will update this logic. Thanks!